include/library/spdm_transport_pcidoe_lib.h

```diff
@@ -62,6 +62,21 @@ libspdm_return_t libspdm_transport_pci_doe_encode_message(
     bool is_requester, size_t message_size, void *message,
     size_t *transport_message_size, void **transport_message);
 
+/**
+ * Encode a DOE discovery message.
+ *
+ * @param  message_size                   size in bytes of the message data buffer.
+ * @param  message                       A pointer to a source buffer to store the message.
+ * @param  transport_message_size         size in bytes of the transport message data buffer.
+ * @param  transport_message             A pointer to a destination buffer to store the transport message.
+ *
+ * @retval RETURN_SUCCESS               The message is encoded successfully.
```

Comment:
Use the `LIBSPDM_STATUS_` macros. We are in the process of converting the old `RETURN_` macros.